demos/python_demos/llm_text_generation/README.md

```diff
@@ -50,46 +104,64 @@ Depending on the use case, `./servable_unary` and `./servable_stream` showcase d
 
 To test unary example:
 ```bash
-docker run -it --rm -p 9000:9000 -v ${PWD}/servable_unary:/workspace -v ${PWD}/model:/model openvino/model_server:py --config_path /workspace/config.json --port 9000
+docker run -it --rm -p 9000:9000 -v ${PWD}/servable_unary:/workspace -v ${PWD}/${SELECTED_MODEL}:/model openvino/model_server:py -e SELECTED_MODEL=${SELECTED_MODEL} --config_path /workspace/config.json --port 9000
+```
+
+To test the streaming example just mount different workspace location from `./servable_stream`.
+It contains modified `model.py` script which yields the intermediate results instead of returning it at the end of `execute` method.
+The `graph.pbtxt` is also modified to include cycle in order to make the Python Calculator run in a loop.  
+
+```bash
+docker run -it --rm -p 9000:9000 -v ${PWD}/servable_stream:/workspace -v ${PWD}/${SELECTED_MODEL}:/model openvino/model_server:py -e SELECTED_MODEL=${SELECTED_MODEL} --config_path /workspace/config.json --port 9000
+```
+
+You can also deploy the quantized model by just changing the model path mounted to the container. For example:
+
+```bash
+docker run -it --rm -p 9000:9000 -v ${PWD}/servable_stream:/workspace -v ${PWD}/${SELECTED_MODEL}_INT8_compressed_weights:/model openvino/model_server:py -e SELECTED_MODEL=${SELECTED_MODEL} --config_path /workspace/config.json --port 9000
```

Comment:
only the last ovms docker container will be tested with following commands. will it work both for client_unary.py and client_stream.py  examples?